postgres/server/init.sls

```diff
@@ -164,13 +164,12 @@ postgresql-pg_hba:
     # Create the empty file before managing to overcome the limitation of check_cmd
     - onlyif: test -f {{ pg_hba_path }} || touch {{ pg_hba_path }}
     # Make a local backup before the file modification
-    - check_cmd: >-
-        salt-call --local file.copy
-        {{ pg_hba_path }} {{ pg_hba_path ~ postgres.config_backup }} remove_existing=true
   {%- endif %}
 {%- else %}
     - replace: False
 {%- endif %}
+    - prereq:
```

Comment:
I guess we must wrap this with `{%- if postgres.config_backup %} ... {%- endif %}` statements, instead of `onlyif` and `check_cmd`, because no need to trigger backup if it was disabled.